src/assets/drizzle/styles/components/_footer.scss

```diff
@@ -0,0 +1,24 @@
+.#{$app-namespace}-c-footer {
```

Comment:
We capitalize the component names. So this would be #{$app-namespace}-c-Footer.

Reply:
Done!